src/Cell.ts

```diff
@@ -12,6 +12,8 @@ export interface CellProperties extends HasColumn, HasItem, HasValue, RegistryMi
 	key: string;
 }
 
+export type CellInterface = WidgetBaseConstructor<CellProperties>;
```

Comment:
I was trying to avoid forcing users to subclass Body itself. As long as it confirms to properties/children, I want any subclass of WidgetBase to work.